core/spec/lib/spree/core/stock_configuration_spec.rb

```diff
@@ -92,4 +92,26 @@
       end
     end
   end
+
+  describe '#inventory_unit_builder_class' do
+    subject { stock_configuration.inventory_unit_builder_class }
+
+    let(:stock_configuration) { described_class.new }
+
+    it "returns Spree::Stock::InventoryUnitBuilder" do
+      is_expected.to be ::Spree::Stock::InventoryUnitBuilder
+    end
+
+    context "with another constant name assigned" do
+      MyInventoryUnitBuilder = Class.new
```

Comment:
It has the side effect of leaving a top-level defined constant for the rest of the test suite. What about removing it with `Object.send(:remove_const, :MyInventoryUnitBuilder)`?